---
layout: archive
title: "People"
---


<p><img style="margin: 9px 15px -10px 0px;" src="/images/group.jpg" width="600" /></p> 
  

### Current Members
<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/shannan_headshot.jpg" width="100" /></p> The Core is led by **[Shannan Ho Sui](shosui@hsph.harvard.edu)**, a Senior Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health, an Affiliated Faculty member of the HSCI, and the Director of the Core. Her expertise includes data integration, data standards, next-generation sequence analysis, and transcriptional gene regulation. She also has experience in pathogen bioinformatics through her postdoctoral work on anti-infective drug discovery. 

She developed the [oPOSSUM](http://opossum.cisreg.ca) system for predicting transcriptional regulators from gene expression data, the [HSCI Stem Cell Discovery Engine](http://discovery.hsci.harvard.edu) to facilitate molecular comparisons of stem cell experiments, and is the Director of the [HSCI Stem Cell Commons](http://stemcellcommons.org/) - an initiative to promote open exchange of stem cell data and analysis tools through a central stem cell data repository and analytical system. You can find her on [Twitter](https://twitter.com/shannanhosui). Shannan has a PhD in Genetics from the University of British Columbia.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/John1.png" width="100" /> </p>**[John Hutchinson](mailto:jhutchin@hsph.harvard.edu)** is a Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health and the Associate Director of the Core. He has a background in academia, and transitioned from the wet lab to computational approaches during his postdoctoral training. He has pursued research in fields ranging from breast cancer biology to the role of epigenetics in allele-specific transcription. He enjoys adapting standard technologies and public data to novel uses.

His work at the Whitehead Institute, Massachusetts General Hospital and Brigham and Women's Hospital familiarized him with many members of the research community in Boston. His background as both a molecular biologist and bioinformatician serve him well in translating basic biological questions into practical bioinformatic approaches.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Radhika.jpg" width="100" /></p> **[Radhika Khetani](mailto:rkhetani@hsph.harvard.edu)** is a Research Scientist in the Department of Biostatistics and the Training Director at the Core, with a focus on developing bioinformatics workshops and training modules of various lengths for the Harvard community [through funding from HMS and (formerly from) the HNDC)](http://bioinformatics.hms.harvard.edu/). 

Her passion is to make genomics data and its analysis accessible to everyone. She has spent the last several years training wet lab biologists to independently perform bioinformatics analyses. She joined the core in Fall 2014 from the High-Performance Biological Computing (HPCBio) group at University of Illinois (UIUC) where she was involved in several research projects, and was in charge of the training program. Her background is in molecular biology, but she switched to computational work as a postdoctoral fellow when she started studying the impact of stress factors on the health of honey bees. Her sequence-based analysis repertoire includes metagenomics, metatranscriptomics, RNA-Seq, genome and transcriptome assembly, in species across the spectrum of life. Radhika has a PhD in Molecular and Cellular Biology from Dartmouth College.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Brad.png" width="100" /></p> **[Brad Chapman](mailto:bchapman@hsph.harvard.edu)** is a Senior Research Scientist in the Department of Biostatistics. He has spent more than 15 years answering biological questions with computational approaches after switching over from a background in wet-lab research. He combines automated high-throughput analysis pipelines with custom visualization and processing tools. By utilizing a wide variety of languages, he strives to maximize code re-use while maintaining the flexibility to answer highly-specific collaborative questions.

Brad is involved in the open source community as a member of the [Open Bioinformatics Foundation](http://www.open-bio.org/wiki/Main_Page), [bcbio](https://github.com/chapmanb/bcbio-nextgen), [Biopython](http://biopython.org/wiki/Main_Page) and [CloudBioLinux](http://cloudbiolinux.org), as well as contributing regularly to freely available [GitHub](http://github.com/chapmanb) and [Bitbucket](http://bitbucket.org/chapmanb/) repositories. He posts about his research on [Blue Collar Bioinformatics](http://bcb.io) and can be found on Twitter as [chapmanb](https://twitter.com/#!/chapmanb). He has a PhD in plant biology from the University of Georgia.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/rory_kirchner_headshot.png" width="100" /></p> **[Rory Kirchner](mailto:kirchner@hsph.harvard.edu)** is a Research Scientist with background in neuroscience and engineering from the Harvard-MIT Health Science and Technology department. He works on creating software to implement best-practice approaches for common informatics analyses and methods development for tackling novel informatics problems. His code can be found on [github](https://github.com/roryk) 
and he blogs at [spliced](https://spliced.ghost.io). 

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Meeta.png" width="100" /></p> **[Meeta Mistry](mailto:mmistry@hsph.harvard.edu)** is a Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health. Her training began in Biochemistry, however postgraduate interests in computational approaches fostered the transition to Bioinformatics. During her graduate career she has pursued various areas of research including the identification of bacterial drug targets, implementing metrics for functional similarity, and cancer proteomics. Her expertise includes complex statistical analysis of large-scale datasets pertaining to postmortem human brain, meta-analysis and genomics of neuropsychiatry. She also has a passion for teaching bioinformatics with experience at the highschool, graduate and postgraduate level. Her current role involves both bioinformatics consulting and training efforts. Meeta has a PhD in Bioinformatics from the University of British Columbia. 

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Lorena2.png" width="100" /></p> **[Lorena Pantano](mailto:lpantano@iscb.org)** is a Research Scientist in the Department of Biostatistics. She graduated with a BSc in Biochemistry, and extended her career in Bioinformatics during her MSc and PhD thesis. Her interests are the application of statistical and mathematical models to large-scale data, mainly (small) RNA-seq data. She likes to think of new visualization concepts to improve the interpretation of complex data providing a dynamic user-data interaction. She is involved on the development/maintenance of [SeqBuster](https://github.com/lpantano/seqbuster), a framework for the analysis of small RNA seq data that integrates Java, Python and R modules. You can find her on [Twitter](https://twitter.com/lopantano). 

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Mary.png" width="100" /></p> **[Mary Piper](mailto:piper@hsph.harvard.edu)** is a Research Associate in the Department of Biostatistics. She serves dual roles as research consultant and bioinformatics trainer, but her primary role is the development and instruction of bioinformatics workshops focused on the analysis of next-generation sequencing data. Her background in scientific curriculum design and instruction began at the University of Michigan, where she taught undergraduates for over five years while completing her PhD in cellular and molecular biology and her MPH in epidemiology. She combined her passions for scientific research and teaching at the Broad Institute of MIT and Harvard as the genomics instructor for MIT’s Minority Introduction to Engineering and Science program where her course focused on cutting-edge biomedical research and bioinformatics analysis. Her exposure to the research at the Broad Institute, MIT, and Harvard, led to her desire to pursue bioinformatics as a career and to share that knowledge with the community.


<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/victor_barrera_headshot.png" width="100" /></p> **[Victor Barrera](mailto:vbarrera@hsph.harvard.edu)**  is a Research Associate in the department of Biostatistics at the Harvard Chan School of Public Health. He graduated with a BSc in Biochemistry and began his thesis in Cancer Epigenetics. During his first PhD year he shifted to Bioinformatics, obtaining a MSc in Bioinformatics and Biostatistics, and designed custom pipeline and analysis for NGS epigenetic data. He is interested in the study of big volumes of data such as next generation sequencing or Big Data in order to find hidden relationships. He is experienced in custom analysis development and automation using Python, R and Bash as his main worktools.


<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/mike_headshot.jpg" width="100" /></p> **[Michael Steinbaugh](mailto:msteinbaugh@hsph.harvard.edu)** is a Research Associate in the department of Biostatistics at the Harvard Chan School of Public Health. He graduated with a BSc in Microbiology, Immunology  and Molecular Genetics from  UCLA and did his Ph.D. at the University of Michigan Medical School where he studied stress resistance and xenobiotic metabolism in slow-aging mice. Mike did his postdoc at the Joslin where he worked in Keith Blackwell’s group on lipid homeostasis and lifespan extension in *C. elegans*. Mike has a keen interest in R and Bioconductor development.

<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/Kayleigh.jpg" width="100" /></p> **[Kayleigh Rutherford](mailto:krutherford@hsph.harvard.edu)**  is a Research Data Analyst in the Department of Biostatistics. She graduated with a BSc in Genetics and Computer Science from the University of Cape Town, and went on to complete a Medical Science Honours in Bioinformatics, with a thesis that focused on computational methods for drug repositioning. She came to Boston to pursue a Masters in Bioinformatics and since graduating, joined the core where she focuses on statistical analysis and visualisations of large-scale data from siRNA and compound screens. 

#### Faculty Director
<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Pete_headshot.png" width="100" /></p>**[Peter Kraft](mailto:pkraft@hsph.harvard.edu)** is a Professor of Epidemiology and Biostatistics at the Harvard T.H. Chan School of Public Health. Dr. Kraft’s research focuses on statistical issues in the design and analysis of genetic association studies, with an emphasis on applications in cancer epidemiology. He has published over 300 articles and was recognized by Thompson Reuters as a highly cited researcher in 2014. Dr. Kraft has also developed and taught courses on introductory genetic epidemiology and “big data” for epidemiologists. Dr. Kraft is the Director of the Harvard Chan Program in Genetic Epidemiology and Statistical Genetics, and Faculty Director of the Harvard Chan Bioinformatics Core (HBC).

Before taking over as Faculty Director of HBC from founding Faculty Advisor Win Hide in August 2014, Dr. Kraft worked closely with the Core to develop pipelines to call and analyze high-depth targeted sequencing in a multi-ethnic sample of 4,600 breast cancer cases and controls. 


#### Administrative support

<p><img style="float: left; margin: 9px 15px -20px 0px;" src="/images/Rachel.jpg" width="100" /><img style="float: left; margin: 9px 15px -20px 0px;" src="/images/megan_scott_headshot.JPG" width="100" /> </p>We are fortunate to have access to dedicated administrative support through the HSPH Department of Biostatistics. **[Rachel Boschetto](mailto:rboschet@hsph.harvard.edu)** is our grants manager and all-around financial guru, and **[Megan Scott](mailto:mescott@hsph.Scottd.edu)** coordinates our events and helps with all planning tasks.

---
 
### Alumni

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/winstonhide.png" width="100" /></p> **[Winston Hide](mailto:iwinhide@sheffield.ac.uk)** created the Core in his role as an Associate Professor of Bioinformatics and Computational Biology in the Department of Biostatistics at Harvard T.H. Chan School of Public Health. Dr Hide is now a Professor of Computational Biology at the  [Sheffield Institute of Translational Neuroscience](https://www.sheffield.ac.uk/neuroscience/staff/hide). He founded and directed the South African National Bioinformatics Institute (SANBI) near Cape Town. With over 20 years of experience in computational biology, his expertise addresses integration of ‘omics data to deliver clinical translation. He uses standardized approaches to disease gene discovery in stem cells and cancer stem cells, host response to pathogens, and complex diseases.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Oliver.JPG" width="100" /></p> **[Oliver Hofmann](mailto:ohofmann@hsph.harvard.edu)** is the Director of Bioinformatics at the [Wolfson Wohl Cancer Research Centre](http://www.gla.ac.uk/researchinstitutes/cancersciences/ics/facilities/wwcrc/) and a Reader in Bioinformatics at the University of [Glasgow](http://www.gla.ac.uk/). Previously the Director of the Core, Oliver continues to consult for the Core as an Affiliated Faculty member of HSPH and the HSCI.  

Oliver worked in a molecular biology wetlab environment before switching to natural language processing, database development and curation as part of his PhD thesis, eventually branching out to sequence clustering and pattern recognition as a visiting researcher at [deCODE Genetics](http://www.decode.com/). His work as a postdoctoral fellow at the [South African National Bioinformatics Institute](http://www.sanbi.ac.za/) focused on data integration, second-generation sequencing data management and network analysis. His recent research focuses on stem cell biology and integrating heterogeneous data sources to provide biological context to results obtained from high throughput experiments, as well as applications of next generation sequencing to cancer genomics. He can be found on [Twitter](https://twitter.com/#!/fiamh) and [blogs about workshops and conferences](http://fiamh.tumblr.com/). Oliver has a PhD in Biochemistry from the University of Cologne, Germany.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Andreas.jpg" width="100" /></p> **[Andreas Sjödin](mailto:andreas.sjodin@umu.se)** was a Research Associate in the Department of Biostatistics and is currently the Deputy Research Director at the FOI in Umea, Sweden. He has been in the field of genomics for more than ten years and closely followed the development of emerging sequencing technologies. The focus of his research has changed from plant transcriptomics through pathogen genome sequencing to human WGS and metagenomics. He is also an expert on forensics of select agents (BSATs) using sequencing technologies. Andreas has a PhD in Cell and Molecular Biology of Plants from [Umeå University](http://www.umu.se). Follow him on [twitter](https://twitter.com/druvus).

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/JWM.png" width="100" /></p> **[John Morrissey](mailto:jmorrissey@fas.harvard.edu)** was part of the [FAS Research Computing team](https://rc.fas.harvard.edu/about-rc/research-computing-staff/) and provided research computing support for the Harvard Chan School of Public Health. Prior to joining Research Computing, John was a Senior Software Engineer at [Frontier Communications](http://frontier.com/) where he was the lead on writing new code and modifying existing code, designing/implementing systems and network infrastructure, performance planning and tuning, and final escalation for production troubleshooting and failure analysis. He works in close collaboration with core staff and external partners to identify and resolve bottlenecks in our workflows. You can follow his latest work on [Twitter](https://twitter.com/johnmorr). John has a BS in Information Technology from Rochester Institute of Technology.
