---
layout: archive
title: "People"
---


<p><img style="margin: 9px 15px -10px 0px;" src="/images/group.jpg" width="600" /></p> 
  

### Current Members
<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/shannan_headshot.jpg" width="100" /></p> The Core is led by **[Shannan Ho Sui](shosui@hsph.harvard.edu)**, a Senior Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health, an Affiliated Faculty member of the HSCI, and the Director of the Core. Her expertise includes data integration, data standards, next-generation sequence analysis, and transcriptional gene regulation. She also has experience in pathogen bioinformatics through her postdoctoral work on anti-infective drug discovery. 

She developed the [oPOSSUM](http://opossum.cisreg.ca) system for predicting transcriptional regulators from gene expression data, the [HSCI Stem Cell Discovery Engine](http://discovery.hsci.harvard.edu) to facilitate molecular comparisons of stem cell experiments, and is the Director of the [HSCI Stem Cell Commons](http://stemcellcommons.org/) - an initiative to promote open exchange of stem cell data and analysis tools through a central stem cell data repository and analytical system. You can find her on [Twitter](https://twitter.com/shannanhosui). Shannan has a PhD in Genetics from the University of British Columbia.

<p><img style="float: left; margin: 9px 15px 1px 0px;" src="/images/John1.jpg" width="100" /> </p>**[John Hutchinson](mailto:jhutchin@hsph.harvard.edu)** is a Senior Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health and helps lead the core as it's Associate Director. He has a background in academia, and transitioned from the wet lab to computational approaches during his postdoctoral training. He has pursued research ranging from studying the role of the Akt serine threonine kinase in breast cancer biology, characterizing the nuclear enriched [lncRNAs NEAT1 and MALAT1](http:/dx.doi.org/10.1186/1471-2164-8-39), helping show evidence of [widespread allele specific expression of autosomal genes](http://dx.doi.org/10.1126/science.1148910)  and developing novel bioinformatic methods to study the [effect of DNA variation on methylation](10.1371/journal.pone.0098464). He enjoys adapting standard technologies and public data to novel uses and designing data visualizations to better illustrate experimental results.

His varied background has familiarized him with many members of the research community in Boston and his deep background as both a molecular biologist and bioinformatician give him unique insight into translating biological questions into practical bioinformatic approaches. He is on [Twitter](https:/twitter.com/ordinator). John has a Ph.D. in Biology from McMaster University.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Radhika.jpg" width="100" /></p> **[Radhika Khetani](mailto:rkhetani@hsph.harvard.edu)** is a Research Scientist in the Department of Biostatistics and the Training Director at the Core, with a focus on developing bioinformatics workshops and training modules of various lengths for the Harvard community [through funding from HMS and (formerly from) the HNDC)](http://bioinformatics.hms.harvard.edu/). 

Her passion is to make genomics data and its analysis accessible to everyone. She has spent the last several years training wet lab biologists to independently perform bioinformatics analyses. She joined the core in Fall 2014 from the High-Performance Biological Computing (HPCBio) group at University of Illinois (UIUC) where she was involved in several research projects, and was in charge of the training program. Her background is in molecular biology, but she switched to computational work as a postdoctoral fellow when she started studying the impact of stress factors on the health of honey bees. Her sequence-based analysis repertoire includes metagenomics, metatranscriptomics, RNA-Seq, genome and transcriptome assembly, in species across the spectrum of life. Radhika has a PhD in Molecular and Cellular Biology from Dartmouth College.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/rory_kirchner_headshot.png" width="100" /></p> **[Rory Kirchner](mailto:kirchner@hsph.harvard.edu)** is a Senior Research Scientist with background in neuroscience and engineering from the Harvard-MIT Health Science and Technology department. He works on creating software to implement best-practice approaches for common informatics analyses and methods development for tackling novel informatics problems. His code can be found on [github](https://github.com/roryk) 
and he blogs at [spliced](https://spliced.ghost.io). 


<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Meeta.png" width="100" /></p> **[Meeta Mistry](mailto:mmistry@hsph.harvard.edu)** is a Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health. Her training began in Biochemistry, however postgraduate interests in computational approaches fostered the transition to Bioinformatics. During her graduate career she has pursued various areas of research including the identification of bacterial drug targets, implementing metrics for functional similarity, and cancer proteomics. Her expertise includes complex statistical analysis of large-scale datasets pertaining to postmortem human brain, meta-analysis and genomics of neuropsychiatry. She also has a passion for teaching bioinformatics with experience at the highschool, graduate and postgraduate level. Her current role involves both bioinformatics consulting and training efforts. Meeta has a PhD in Bioinformatics from the University of British Columbia. 

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Mary.png" width="100" /></p> **[Mary Piper](mailto:piper@hsph.harvard.edu)** is a Research Associate in the Department of Biostatistics. She serves dual roles as research consultant and bioinformatics trainer, but her primary role is the development and instruction of bioinformatics workshops focused on the analysis of next-generation sequencing data. Her background in scientific curriculum design and instruction began at the University of Michigan, where she taught undergraduates for over five years while completing her PhD in cellular and molecular biology and her MPH in epidemiology. She combined her passions for scientific research and teaching at the Broad Institute of MIT and Harvard as the genomics instructor for MIT’s Minority Introduction to Engineering and Science program where her course focused on cutting-edge biomedical research and bioinformatics analysis. Her exposure to the research at the Broad Institute, MIT, and Harvard, led to her desire to pursue bioinformatics as a career and to share that knowledge with the community.

<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/victor_barrera_headshot.png" width="100" /></p> **[Victor Barrera](mailto:vbarrera@hsph.harvard.edu)**  is a Research Associate in the department of Biostatistics at the Harvard Chan School of Public Health. He graduated with a BSc in Biochemistry and began his thesis in Cancer Epigenetics. During his first PhD year he shifted to Bioinformatics, obtaining a MSc in Bioinformatics and Biostatistics, and designed custom pipeline and analysis for NGS epigenetic data. He is interested in the study of big volumes of data such as next generation sequencing or Big Data in order to find hidden relationships. He is experienced in custom analysis development and automation using Python, R and Bash as his main worktools.


<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/james.png" width="100" /></p> **[James Billingsley](mailto:jbillingsley@hsph.harvard.edu)** is a Research Scientist in the Department of Biostatistics at the Harvard Chan School of Public Health. He has a PhD in Immunology and developed his interests in high dimension computational methods during his postdoctoral training. Prior to joining the Core, James worked as a Bioinformatics Analyst at the Yerkes Non-human Primate Genomics Core at Emory University. His primary interests at the Chan Bioinformatics Core are focused on single-cell RNA-seq, and his additional research interests include characterizing correlates of protection in non-human primate models of HIV vaccines, and understanding mechanisms maintaining HIV latency. 


<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/zhu_zhuo.jpg" width="100" /></p> **[Zhu Zhuo](mailto:zzhuo@hsph.harvard.edu)** is a Research Associate in the Department of Biostatistics at the Harvard Chan School of Public Health. She has a PhD degree in Animal and Food Sciences from the University of Delaware. During her PhD, she worked on analyses of differential gene expression and allele specific gene expression in chickens. Her postdoctoral training focused on de novo genome assembly in plants and supporting undergraduate education in Genetics and Genomics. Zhu’s role at the Harvard Chan Bioinformatics Core is to provide Bioinformatics consulting.


<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/sergey_headshot.jpg" width="100" /></p> **[Sergey Naumenko](mailto:snaumenko@hsph.harvard.edu)** is a Research Associate in the Department of Biostatistics at the Harvard Chan School of Public Health. Sergey graduated with BSc/MSc in Computer Science from Kharkiv National University of Radio and Electronics (Kharkiv, Ukraine), defended PhD thesis in Mathematical Biology and Bioinformatics in the joint laboratory of Evolutionary Genomics and Molecular Evolution (Moscow State University/Institute for Information Transmission Problems, Moscow, Russia), and went through a postdoctoral training at the Hospital for Sick Children (Toronto, Canada). Sergey is performing RNA-seq analyses and is helping to maintain the bcbio ecosystem. [Publications](https://scholar.google.ca/citations?user=SeL177gAAAAJ&hl=en), [GitHub](https://github.com/naumenko-sa), [Twitter](https://twitter.com/SergeyNaumenko6).

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/jay.jpg" width="100" /></p> **[Joon Yoon](mailto:jyoon@hsph.harvard.edu)** is a Research Associate in the Department of Biostatistics at the Harvard Chan School of Public Health. Joon obtained his MSc and PhD degrees in Bioinformatics from the Seoul National University where he defended his thesis on custom workflows and visualizations for transcriptomic analyses (using R and python scripting). During his Ph.D. Joon joined the bioinformatics solution company C&K Genomics' RNA-seq team as a research consultant where he helped set update their  bulk RNA-seq analysis workflows by switching out outdated protocols and expanding on types of analyses. While there he also participated in the T2D international consortium and several other RNA-seq projects. 


<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/jihe.png" width="100" /></p> **[Jihe Liu](mailto:liujihe.2011@gmail.com)** is a Research Associate in the Department of Biostatistics at the Harvard Chan School of Public Health. Jihe obtained his BS degree in Bioinformatics from Huazhong University of Science and Technology, and PhD degree in Chemistry (with focus on chemical biology and synthetic biology) from University of Pittsburgh. He received postdoctoral training at The Jackson Laboratory, where he worked on genomic analysis of Glioblastoma using short-read and long-read sequencing technologies. Jihe’s role at the core is to instruct bioinformatics workshops, develop training materials, and support selected data analysis projects.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/preeti.jpg" width="100" /></p> **[Preetida Bhetariya](mailto:pbhetariya@hsph.harvard.edu)**
Dr. Preetida J Bhetariya is a Research Associate in the Department of Biostatistics. She received her PhD in Biotechnology from Institute of Genomics and Integrative Biology, India where she studied Aspergillus genomics for secondary metabolites. Along with molecular and functional genomics approaches, she developed a diagnostic test for fungal (Aspergillus) detection from clinical samples. She continued pursuing her computational genomics interests during her post-doctoral training at University of Utah where she used RNA-seq to study microbial interactions in Multiple Sclerosis and characterized human gene expression patterns in MS and other autoimmune diseases. Following her passion for diagnostic research, she moved to Dr. Gabor Marth’s lab where she studied cancer genomics; developing a  bioinformatics pipeline for low-frequency variant calling for circulating tumor DNA from liquid biopsies. She is an expert in designing custom custom analysis workflows in Python and R. She supports the data analytics of projects involving single cell genomics and cancer genomics at the core.

[Github]( https://github.com/preetida),
[Google Scholar](https://scholar.google.com/citations?user=tpmwIvMAAAAJ&hl=en&oi=ao), [Twitter](https://twitter.com/preetiaahir)

 
<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/ilya.jpg" width="100" /></p> **[Ilya Sytchev](mailto:isytchev@hsph.harvard.edu)** is a Senior Bioinformatics Software Developer in the Department of Biostatistics. He was one of the founders of [OpenWetWare](https://openwetware.org/) - an online community for biologists and is currently helping build and maintain the [HSCI Stem Cell Commons](https://stemcellcommons.org/). He has a BS in Computer Science from University of Massachusetts Boston and a MS in Bioinformatics from Northeastern University.



#### Faculty Director
<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Pete_headshot.png" width="100" /></p>**[Peter Kraft](mailto:pkraft@hsph.harvard.edu)** is a Professor of Epidemiology and Biostatistics at the Harvard T.H. Chan School of Public Health. Dr. Kraft’s research focuses on statistical issues in the design and analysis of genetic association studies, with an emphasis on applications in cancer epidemiology. He has published over 300 articles and was recognized by Thompson Reuters as a highly cited researcher in 2014. Dr. Kraft has also developed and taught courses on introductory genetic epidemiology and “big data” for epidemiologists. Dr. Kraft is the Director of the Harvard Chan Program in Genetic Epidemiology and Statistical Genetics, and Faculty Director of the Harvard Chan Bioinformatics Core (HBC).

Before taking over as Faculty Director of HBC from founding Faculty Advisor Win Hide in August 2014, Dr. Kraft worked closely with the Core to develop pipelines to call and analyze high-depth targeted sequencing in a multi-ethnic sample of 4,600 breast cancer cases and controls. 


#### Administrative support

<p><img style="float: left; margin: 9px 15px -20px 0px;" src="/images/Rachel.jpg" width="100" /><img style="float: left; margin: 9px 15px -20px 0px;" src="/images/maria.png" width="100" /> </p>We are fortunate to have access to dedicated administrative support. **[Rachel Boschetto](mailto:rboschet@hsph.harvard.edu)** is our grants manager and all-around financial guru, and **[Maria Simoneau](mailto:msimoneau@hsph.harvard.edu)** is our Project Coordinator, helping with grant support, record keeping, reporting, planning and event coordination.

---
 
### Alumni

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/winstonhide.png" width="100" /></p> **[Winston Hide](mailto:iwinhide@sheffield.ac.uk)** created the Core in his role as an Associate Professor of Bioinformatics and Computational Biology in the Department of Biostatistics at Harvard T.H. Chan School of Public Health. Dr Hide is currently an [Associate Professor of Pathology at the Beth Israel Deaconess Medical Center](https://www.bidmc.org/research/research-by-department/pathology/laboratories/winston-hide-lab) where he also co-directs the [Non-coding RNA Core Facility](https://www.bidmc.org/research/core-facilities/ncrna-core). He founded and directed the South African National Bioinformatics Institute (SANBI) near Cape Town. With over 20 years of experience in computational biology, his expertise addresses integration of ‘omics data to deliver clinical translation. He uses standardized approaches to disease gene discovery in stem cells and cancer stem cells, host response to pathogens, and complex diseases.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Oliver.JPG" width="100" /></p> **[Oliver Hofmann](mailto:ohofmann@hsph.harvard.edu)** is the Director of Bioinformatics at the [Wolfson Wohl Cancer Research Centre](http://www.gla.ac.uk/researchinstitutes/cancersciences/ics/facilities/wwcrc/) and a Reader in Bioinformatics at the University of [Glasgow](http://www.gla.ac.uk/). Previously the Director of the Core, Oliver continues to consult for the Core as an Affiliated Faculty member of HSPH and the HSCI.  

Oliver worked in a molecular biology wetlab environment before switching to natural language processing, database development and curation as part of his PhD thesis, eventually branching out to sequence clustering and pattern recognition as a visiting researcher at [deCODE Genetics](http://www.decode.com/). His work as a postdoctoral fellow at the [South African National Bioinformatics Institute](http://www.sanbi.ac.za/) focused on data integration, second-generation sequencing data management and network analysis. His recent research focuses on stem cell biology and integrating heterogeneous data sources to provide biological context to results obtained from high throughput experiments, as well as applications of next generation sequencing to cancer genomics. He can be found on [Twitter](https://twitter.com/#!/fiamh) and [blogs about workshops and conferences](http://fiamh.tumblr.com/). Oliver has a PhD in Biochemistry from the University of Cologne, Germany.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Brad.png" width="100" /></p> **[Brad Chapman](mailto:bchapman@hsph.harvard.edu)** was a Senior Research Scientist in the Department of Biostatistics. Currently working at [Gingko Bioworks](https://www.ginkgobioworks.com/), he has spent more than 15 years answering biological questions with computational approaches after switching over from a background in wet-lab research. He combines automated high-throughput analysis pipelines with custom visualization and processing tools. By utilizing a wide variety of languages, he strives to maximize code re-use while maintaining the flexibility to answer highly-specific collaborative questions.

Brad is involved in the open source community as a member of the [Open Bioinformatics Foundation](http://www.open-bio.org/wiki/Main_Page), [bcbio](https://github.com/chapmanb/bcbio-nextgen), [Biopython](http://biopython.org/wiki/Main_Page) and [CloudBioLinux](http://cloudbiolinux.org), as well as contributing regularly to freely available [GitHub](http://github.com/chapmanb) and [Bitbucket](http://bitbucket.org/chapmanb/) repositories. He posts about his research on [Blue Collar Bioinformatics](http://bcb.io) and can be found on Twitter as [chapmanb](https://twitter.com/#!/chapmanb). He has a PhD in plant biology from the University of Georgia.

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Lorena2.png" width="100" /></p> **[Lorena Pantano](mailto:lpantano@iscb.org)**  was a Research Scientist in the Department of Biostatistics. Currewntly at [eGenesis](https://www.egenesisbio.com/), she graduated with a BSc in Biochemistry, and extended her career in Bioinformatics during her MSc and PhD thesis. Her interests are the application of statistical and mathematical models to large-scale data, mainly (small) RNA-seq data. She likes to think of new visualization concepts to improve the interpretation of complex data providing a dynamic user-data interaction. She is involved on the development/maintenance of [SeqBuster](https://github.com/lpantano/seqbuster), a framework for the analysis of small RNA seq data that integrates Java, Python and R modules. You can find her on [Twitter](https://twitter.com/lopantano). 

<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/mike_headshot.jpg" width="100" /></p> **[Michael Steinbaugh](mailto:msteinbaugh@hsph.harvard.edu)** was a Research Associate in the department of Biostatistics at the Harvard Chan School of Public Health. He graduated with a BSc in Microbiology, Immunology  and Molecular Genetics from  UCLA and did his Ph.D. at the University of Michigan Medical School where he studied stress resistance and xenobiotic metabolism in slow-aging mice. Mike did his postdoc at the Joslin where he worked in Keith Blackwell’s group on lipid homeostasis and lifespan extension in *C. elegans*. Mike has a keen interest in R and Bioconductor development. Mike left the core for a position at [Constellation Pharmaceuticals](https://www.constellationpharma.com/),  where he puts his expertise in single cell RNA sequencing to good use..

<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/Kayleigh.jpg" width="100" /></p> **[Kayleigh Rutherford](mailto:krutherford@hsph.harvard.edu)**  was a Research Data Analyst in the Department of Biostatistics. She graduated with a BSc in Genetics and Computer Science from the University of Cape Town, and went on to complete a Medical Science Honours in Bioinformatics, with a thesis that focused on computational methods for drug repositioning. She came to Boston to pursue a Masters in Bioinformatics and since graduating, joined the core where she focuses on statistical analysis and visualisations of large-scale data from siRNA and compound screens. She has since moved on to a Computational Biology position at Memorial Sloan Kettering. 

<p><img style="float: left; margin: 9px 15px 0px 0px;" src="/images/laura_lin.jpg" width="100" /></p> **[Yufei Lin](mailto:laura21@bu.edu)** was an Intern in the core and came to us from the Boston University where she was pursuing a Masters in Statistical Practice, which fostered her interest in the application of statistics to  biology. Yufei has an extensive and varied background in data science and analysis and a Bachelor of Management in Electronic Commerce.  Her working experience in a neuroscience lab and a biomedical genetics lab makes her determined to pursue bioinformatics as a career, specifically in genomics studies, where she is most interested in studying next generation sequencing data to find relationships between the genome and disease. She is experienced in statistical analysis and visualization of large-scale data using R. 


<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/Andreas.jpg" width="100" /></p> **[Andreas Sjödin](mailto:andreas.sjodin@umu.se)** was a Research Associate in the Department of Biostatistics and is currently the Deputy Research Director at the FOI in Umea, Sweden. He has been in the field of genomics for more than ten years and closely followed the development of emerging sequencing technologies. The focus of his research has changed from plant transcriptomics through pathogen genome sequencing to human WGS and metagenomics. He is also an expert on forensics of select agents (BSATs) using sequencing technologies. Andreas has a PhD in Cell and Molecular Biology of Plants from [Umeå University](http://www.umu.se). Follow him on [twitter](https://twitter.com/druvus).

<p><img style="float: left; margin: 9px 15px -10px 0px;" src="/images/JWM.png" width="100" /></p> **[John Morrissey](mailto:jmorrissey@fas.harvard.edu)** was part of the [FAS Research Computing team](https://rc.fas.harvard.edu/about-rc/research-computing-staff/) and provided research computing support for the Harvard Chan School of Public Health. Prior to joining Research Computing, John was a Senior Software Engineer at [Frontier Communications](http://frontier.com/) where he was the lead on writing new code and modifying existing code, designing/implementing systems and network infrastructure, performance planning and tuning, and final escalation for production troubleshooting and failure analysis. He works in close collaboration with core staff and external partners to identify and resolve bottlenecks in our workflows. You can follow his latest work on [Twitter](https://twitter.com/johnmorr). John has a BS in Information Technology from Rochester Institute of Technology.
