---
layout: archive
title: "Resources"
---

----

In addition to the material here, our teaching team maintains a list of useful [bioinformatics resources](https://wiki.harvard.edu/confluence/display/hbctraining/Resources).

### Knowledgebase
Our team collaborates on an eclectic list of useful bioinformatic tips and tricks in our [Knowledgebase](http://bioinformatics.sph.harvard.edu/knowledgebase)


### Software

#### bcbio.nextgen

Our NGS analyses rely on the [bcbio.nextgen](https://github.com/chapmanb/bcbio-nextgen) framework, a community-developed NGS workflow that comes with [full documentation](https://bcbio-nextgen.readthedocs.org/en/latest/), is open source under the [MIT license](http://opensource.org/licenses/mit-license.html), and is in use at over a dozen sites internationally. It is installed at both the FAS and HMS Research Computing environments and provides researchers with best practice workflows for [exome / whole-genome sequencing](https://bcbio-nextgen.readthedocs.org/en/latest/contents/pipelines.html#variant-calling) (built around GATK 3.0 and FreeBayes) , [RNA-Seq](https://bcbio-nextgen.readthedocs.org/en/latest/contents/pipelines.html#rna-seq) (TopHat2/STAR, [Sailfish/Salmon](https://bcbio-nextgen.readthedocs.io/en/latest/contents/pipelines.html#fast-rna-seq), edgeR/DESeq2/limma), and most recently, [small RNA-seq](https://bcbio-nextgen.readthedocs.io/en/latest/contents/pipelines.html#smallrna-seq), [single-cell RNA-seq](https://bcbio-nextgen.readthedocs.io/en/latest/contents/pipelines.html#single-cell-rna-seq) and the intial steps of [ChIP-seq](https://bcbio-nextgen.readthedocs.io/en/latest/contents/pipelines.html#chip-seq). Take a look at some of the key blog posts around bcbio:

* [Scaling NGS pipelines for WGS](https://bcbio.wordpress.com/2013/05/22/scaling-variant-detection-pipelines-for-whole-genome-sequencing-analysis/)
* [Ensembl variant calling](https://bcbio.wordpress.com/2013/10/21/updated-comparison-of-variant-detection-methods-ensemble-freebayes-and-minimal-bam-preparation-pipelines/)
* [Docker for easier distribution](https://bcbio.wordpress.com/)
* [Introducing RNA-Seq to bcbio](http://spliced.ghost.io/automated-comparison-of-rna-seq-differential-expression-calls/)

[More presentations about bcbio](https://bcbio-nextgen.readthedocs.io/en/latest/contents/presentations.html)

#### BioGrids

If you are working on the HMS Orchestra cluster and need a piece of bioinformatics software, we recommend you look at the software available through BioGrids project, a [software stack for bioinformatics available on the HMS Orchestra cluster](http://www.biogrids.org/)

#### Conda

We also recommend [conda](https://conda.io/docs/index.html) as an easy way to install and run software in a shared environment without worrying about dependencies. Many general use packages can be found on [Anaconda Cloud](https://anaconda.org/) and bioinformatics software can be found through [bioconda](https://bioconda.github.io/).

----

### Computational and Data Storage Resources

#### Research Computing 

We work closely with and use the computational resources of both:

-  the Odyssey high performance cluster at the [FAS Research Computing Center](http://rc.fas.harvard.edu/)
-  the Orchestra high performance cluster at [Harvard Medical School Research Computing](https://rc.hms.harvard.edu)

We also use a variety of cloud architectures including [Amazon AWS](https://aws.amazon.com/) and [Microsoft Azure](https://azure.microsoft.com/en-us/)](https://azure.microsoft.com/en-us/)

#### Data Management

Data storage needs increase by the day. To help to figure out how to keep storage costs manageble, we contribute to an important initiative at HMS to develop and propagate good data management practices and resources across the Harvard community. For details and tips on managing your data, see the [HMS data management page](http://datamanagement.hms.harvard.edu/) 

----

### Next Generation Sequencing

We work with data from any sequencing core but currently work most closely with the:

- [Biopolymers facility](https://genome.med.harvard.edu/)

the

- [Molecular Biology Core facility at the Dana Farber Cancer Institute](https://mbcf.dfci.harvard.edu/)

and the

- [Bauer Core Facility at FAS](https://bauercore.fas.harvard.edu/








